tools/cldr-apps/js-unittest/pretty.sh

```diff
@@ -0,0 +1,2 @@
+npx prettier --write tools/cldr-apps/src/main/webapp/js/new/*.js
+npx prettier --write tools/cldr-apps/js-unittest/*.js
```

Comment:
this is fine, but can also be done within npm with no .sh needed (could be called from maven)

```diff
diff --git a/tools/cldr-apps/js/package.json b/tools/cldr-apps/js/package.json
index 22c6369078..8cafacf354 100644
--- a/tools/cldr-apps/js/package.json
+++ b/tools/cldr-apps/js/package.json
@@ -6,7 +6,8 @@
   "scripts": {
     "test": "exit 0",
     "lint": "exit 0",
-    "build": "webpack"
+    "build": "webpack",
+    "prettier": "npx prettier --write ../src/main/webapp/js/new/*.js ../js-unittest/*.js"
   },
   "repository": {
     "type": "git",
```

Reply:
sounds good -- how would you then invoke it manually?

Reply:
@btangmu 

`npm run prettier`  in the `js` dir